crates/nu-command/src/strings/detect_columns.rs

```diff
@@ -82,8 +82,87 @@ impl Command for DetectColumns {
             },
             Example {
                 description: "Splits a multi-line string into columns with headers detected",
-                example: "$'c1 c2 c3(char nl)a b c' | detect columns",
-                result: None,
+                example: "\"c1 c2 c3 c4 c5\na b c d e\" | detect columns",
```

Comment:
I am not sure if this get's rendered properly in our `help` or the `F1` help menu. Not sure if we take multiple lines properly into account.

Reply:
I think you're right @sholderbach. I believe it's better like this for examples.
```suggestion
                example: "$'c1 c2 c3 c4 c5(char nl)a b c d e' | detect columns",
```

Reply:
the string interpolation can not be handled right by the strings::test_example and can not be test passed, so I modified by the escape string and the example display normal